### ISSUE TITLE
MIME type "text/markdown; charset=utf-8" is not supported (parsed as "text/markdown;" instead)

### ISSUE THREAD

**Issue by JohnScience** [google/evcxr#232]
I tried to provide as much information as possible to `evcxr` and tried to supply markdown in the form of

```
EVCXR_BEGIN_CONTENT text/markdown; charset=utf-8
$$
...
$$
EVCXR_END_CONTENT
```

It could be reasonable to have info somewhere about supported MIME types.

**Comment by davidlattimore** [comment]
You're right that a list of supported mime types is a good idea. Evcxr just passes the mime type through to Jupyter notebook - so I guess whatever it supports. I'm slightly surprised that it doesn't seem to support markdown, given that it lets users write markdown in notebook cells.

The mime types that I know work are text/plain, text/html and at least some types of images, e.g. image/png.

**Comment by JohnScience** [comment]
@davidlattimore `text/markdown` is supported. However, `text/markdown; charset=utf-8` is not.
